role-management-commands/list-temproles.md

```diff
@@ -1,6 +1,15 @@
 # list-temproles
 
-<TODO: describe>
+This command allows you to view all active temporary roles and paused roles in the server. It will display the user,
+role, and the time remaining for each temporary/paused role.
+
+You can also specify a target to filter the results to only show the temporary roles of a user or the users of a role.
+For further information on the target parameter, refer to the [Options](#options) section.
+
+{% hint style="info" %}
+For Information on how to assign a temporary role to a user, refer to the [temprole](../role-management-commands/temprole.md#temprole) command and
```

Comment:
```suggestion
For information on how to assign a temporary role to a user, refer to the [`temprole`](../role-management-commands/temprole.md#temprole) command and
```

Command names in `inline code`, nouns in English aren't capitalized as in German